com.playeveryware.eos/Documentation~/contributions.md

```diff
@@ -59,6 +59,10 @@ A successful build will place the correct binaries in the proper locations for U
 
 See [standards.md](/com.playeveryware.eos/Documentation~/standards.md).
 
+## Unit Testing
+
+See [our documentation on unit testing](/com.playeveryware.eos/Documentation~/unit-testing.md)
```

Comment:
Add punctuation mark at the end of this and the same sections when added to other documents.